config/jobs/kubernetes-sigs/kueue/kueue-presubmits-main.yaml

```diff
@@ -58,7 +58,7 @@ presubmits:
           limits:
             cpu: "4"
             memory: "16Gi"
-  - name: pull-kueue-test-e2e-main-1-24
```

Comment:
I think we can keep it for this release. 1.24 will be in support when we release kueue 0.4.

Reply:
+1